subsys/net/ip/CMakeLists.txt

```diff
@@ -39,12 +39,13 @@ zephyr_library_sources_ifdef(CONFIG_NET_TCP          tcp.c)
 zephyr_library_sources_ifdef(CONFIG_NET_TEST_PROTOCOL           tp.c)
```

Comment:
Typo in commit "net: lib: sockets: improve scalability through **addtl** hidden var" subject.
Perhaps we could say here `net: lib: sockets: improve scalability through a hidden var` instead

Reply:
Fixed.